Sources/StreamChatSwiftUI/ChatChannelList/SearchResultsView.swift

```diff
@@ -52,6 +52,7 @@ struct SearchResultsView<Factory: ViewFactory>: View {
                     }
                 }
             }
+            .background(Color(colors.background))
```

Comment:
That little bugger tried to sneak in again.

```suggestion
```

Reply:
Good catch, thanks!